cloud/linode/loadbalancers.go

```diff
@@ -24,7 +26,11 @@ import (
 	"github.com/linode/linodego"
 )
 
-var errNoNodesAvailable = errors.New("no nodes available for nodebalancer")
+var (
+	errNoNodesAvailable = errors.New("No nodes available for nodebalancer")
+	errInvalidFWConfig  = errors.New("Specify either an allowList or a denyList for a firewall")
+	errTooManyFirewalls = errors.New("Too many firewalls attached to a nodebalancer")
```

Comment:
how do we even get to the too many firewalls error, just if a user attached a firewall to the nodebalancer by hand? should we offer additional error messages for which firewalls we found so they can go clean up themselves if this happens?

Reply:
Yes, we'd get there if the user attaches multiple firewalls by hand - I wanted to be defensive here and not remove stuff the ccm didn't create. I will add a line printing the IDs so the user can investigate